api/src/main/resources/messages_fr.properties

```diff
@@ -89,7 +89,9 @@ patientflags.errors.noPatientIdCriteria=Critère doit avoir '*.patient_id'
 patientflags.errors.noTag=Le nom de l''onglet est requis
 patientflags.errors.noEvaluator=Sélectionnez un évaluateur
 patientflags.errors.nameTooLong=Un nom ne peut comporter que 255 caractères
+patientflags.errors.noUniqueName=Le nom de l'indicateur existe d�j�. Changer pour un autre nom
```

Comment:
Do you mind looking into the above strange characters after `existe`?

Reply:
I've just copied values from translator because I don't know French. But please take a look at whole this file, there are a lot of such characters